ROADMAP.md

```diff
@@ -1,6 +1,14 @@
 # Roadmap
 
-## Release 3.0 (Completed 30 June 2018)
+## Release 3.1 "Harder, Better, Faster, Stronger" (Target Release Date: 30 September 2018)
+
+- Static analysis of roles during import/upload using ansible-lint and additional Galaxy best practice rules
+- Roles quality scoring improved the search result ranking and presentation to make it even easier to find the best content
+- Introduce versioned content
```

Comment:
What does this mean, exactly?

Reply:
It means galaxy content can have version numbers that will be enforced and managed.